src/components/CytoscapeGraph/ContextMenu/NodeContextMenu.tsx

```diff
@@ -171,9 +171,9 @@ export const getOptions = (
 };
 
 const mapStateToProps = (state: KialiAppState) => ({
-  jaegerIntegration: state.jaegerState ? state.jaegerState.integration : false,
+  jaegerIntegration: state.jaegerState ? state.jaegerState.enabled : false,
   namespaceSelector: state.jaegerState ? state.jaegerState.namespaceSelector : true,
```

Comment:
I prefer to move this to jaegerEnabled if we are going to remove the integration check.